blog/2024-04-13-deprecate_async_track_state_change.md

```diff
@@ -0,0 +1,40 @@
+---
+author: J. Nick Koston
+authorURL: https://github.com/bdraco
+title: "Replacing `async_track_state_change` with `async_track_state_change_event`"
+---
+
+`async_track_state_change` is deprecated and will be removed in Home Assistant 2025.5. `async_track_state_change_event` should be used instead.
+
+`async_track_state_change` always creates a top-level listener for `EVENT_STATE_CHANGED`, which would have to reject all state changes that did not match the desired entities. This design presented a performance problem when there were many integrations using `async_track_state_change`. `async_track_state_change` has been phased out in `core` since the introduction of `async_track_state_change`, with the last instance being removed in 2024.5.
```

Comment:
```suggestion
`async_track_state_change` always creates a top-level listener for `EVENT_STATE_CHANGED`, which would have to reject all state changes that did not match the desired entities. This design presented a performance problem when there were many integrations using `async_track_state_change`. `async_track_state_change` has been phased out in `core` since the introduction of `async_track_state_change_event`, with the last instance being removed in 2024.5.
```

Reply:
Thanks